package.json

```diff
@@ -9,25 +9,30 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@prisma/client": "3.0.2",
-    "next": "11.1.2",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@prisma/client": "^2.31.0-dev.9",
+    "bcrypt": "^5.0.1",
+    "next": "9.5.5",
+    "next-auth": "^3.29.0",
+    "react": "16.14.0",
+    "react-dom": "16.14.0"
   },
   "devDependencies": {
+    "@types/bcrypt": "^5.0.0",
+    "@types/next-auth": "3.1.14",
+    "@types/node": "14.14.6",
     "@tailwindcss/jit": "^0.1.18",
-    "@types/react": "17.0.20",
+    "@types/react": "16.9.56",
```

Comment:
Why did you downgrade to react 16?

Reply:
There was a problem with the tailwindcss, postcss and autoprefixer if the versions were to be the latest so as node. since I had faced the exact problem before on the previous task, I used exact versions of the previous ones that worked